Gemfile.lock

```diff
@@ -5,6 +5,16 @@ GIT
   specs:
     ruby-saml-idp (0.3.2)
 
+GIT
+  remote: https://github.com/thoughtbot/action_dispatch-testing-integration-capybara.git
```

Comment:
The reason this hasn't been released as a standalone gem through RubyGems is because it relies on parts of the Rails API not explicitly intended for public use. 

There is ongoing work to make changes to Rails to address this, though.